pkg/diagnose/diagnosis/loader.go

```diff
@@ -93,6 +94,12 @@ type Diagnosis struct {
 	Remediation string
 	// run-time
 	RawError string
+	// Status code
+	StatusCode int
+	// endpoint URL
+	URL string
+	// Name of the endpoint
+	EndpointName string
```

Comment:
If we want to use the field name we will need to refacto a bit diagnose code, since as I talked above name field is just use to store a whole string that does not fit to what we want to display